cli/cmd/edges.go

```diff
@@ -0,0 +1,297 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/linkerd/linkerd2/controller/api/util"
+	pb "github.com/linkerd/linkerd2/controller/gen/public"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type indexedEdgeResults struct {
+	ix   int
+	rows []*pb.Edge
+	err  error
+}
+
+func newCmdEdges() *cobra.Command {
+	options := newStatOptions()
+
+	cmd := &cobra.Command{
+		Use:   "edges [flags] (RESOURCETYPE)",
+		Short: "Display connections between resources, and the identity of their associated Linkerd proxies (if known)",
```

Comment:
i'd shorten this up. while we're not perfectly adhering to 80 chars for the default cli help output, this line is significantly longer than the other commands.

maybe something like:
```
Display connections between resources, and Linkerd proxy identities
```

```
Available Commands:
  check       Check the Linkerd installation for potential problems
  completion  Output shell completion code for the specified shell (bash or zsh)
  dashboard   Open the Linkerd dashboard in a web browser
  edges       Display connections between resources, and the identity of their associated Linkerd proxies (if known)
  endpoints   Introspect Linkerd's service discovery state
  get         Display one or many mesh resources
  help        Help about any command
  inject      Add the Linkerd proxy to a Kubernetes config
  install     Output Kubernetes configs to install Linkerd
  install-cni Output Kubernetes configs to install Linkerd CNI (experimental)
  install-sp  Output Kubernetes configs to install Linkerd Service Profiles
  logs        Tail logs from containers in the Linkerd control plane
  metrics     Fetch metrics directly from Linkerd proxies
  profile     Output service profile config for Kubernetes
  routes      Display route stats
  stat        Display traffic stats about one or many resources
  tap         Listen to a traffic stream
  top         Display sorted information about live traffic
  uninject    Remove the Linkerd proxy from a Kubernetes config
  upgrade     Output Kubernetes configs to upgrade an existing Linkerd control plane
  version     Print the client and server version information
```

Reply:
Thanks, I was trying to figure out how to express that simply - your suggestion is much more clear.